cmd/agi/main.go

```diff
@@ -30,7 +30,7 @@ import (
 const (
 	versionPrefix = `version "`
 	googleInfix   = "-google-"
-	minJavaMajor  = 11
+	minJavaMajor  = 17
```

Comment:
You can't do that without also updating the build scripts to upgrade the bundled JRE. For Windows and MacOS we bundle the JRE with the application, look at the build scripts in the kokoro directory. For Linux, the .deb file should depend on a package providing the version of the JRE we need.